### DISCUSSION TITLE
Where is the example?

### DISCUSSION THREAD

**Discussion by Larsplays** [cefsharp/CefSharp#4000] [Q&A]
Im not sure but i think there was an Code example somewhere and im searching for it but cant find it..

**Comment by amaitland** [comment]
If you are new to the project then I'd suggest you start with https://github.com/cefsharp/CefSharp/wiki/Quick-Start

There are detailed code examples contained within the project source and https://github.com/cefsharp/CefSharp.MinimalExample provides a basic example.
